lib/gtfs_df/base_gtfs_table.rb

```diff
@@ -15,7 +15,9 @@ def initialize(input)
           df = Polars.read_csv(input, infer_schema_length: 0)
           dtypes = self.class::SCHEMA.slice(*df.columns)
 
-          df
+          cleaned_df = df.rename(->(col) { col.strip })
+
+          cleaned_df
```

Comment:
We could append the rename on the initial assignment

```suggestion
          df = Polars.read_csv(input, infer_schema_length: 0).rename(->(col) { col.strip })

          dtypes = self.class::SCHEMA.slice(*df.columns)
          df
```